src/openms/include/OpenMS/APPLICATIONS/MapAlignerBase.h

```diff
@@ -84,7 +84,7 @@ class TOPPMapAlignerBase :
     Param params;
     params.setValue("type", default_model, "Type of model");
     // TODO: avoid referring to each TransformationModel subclass explicitly
-    StringList model_types = ListUtils::create<String>("linear,b_spline,lowess,interpolated");
+    std::vector<std::string> model_types = ListUtils::create<std::string>("linear,b_spline,lowess,interpolated");
```

Comment:
maybe better to replace these constructs with:
= {"linear_b", "b_spline", "lowess", "interpolated"}